### ISSUE TITLE
[Bug]: Nodes and some context menu elements disappear (Kubuntu)

### ISSUE THREAD

**Issue by vibl** [Ironclad/rivet#139]
### What happened?

Nodes become immediately invisible after adding them (regardless of the node type). They only appear briefly (< 1 second) on mouse-over, and then become invisible again.

If I manage to move a node below the left sidebar, it appears permanently (but blurred).

Also, the text of the tabs of the settings modal is invisible ("general", "OpenAI", "plugins" according to the doc). Same for the text of the items of the context menu of the left sidebar.

Feel free to ask me technical questions or suggest technical procedures: I'm a web developer.

### What was the expected functionality?

Nodes should be visible.

### Describe your environment

Kubuntu 23.04
Node.js v20.6.1
https://github.com/Ironclad/rivet/releases/download/app-v1.1.0/rivet_1.1.0_amd64.AppImage

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by abrenneke** [comment]
Thanks @vibl! This is some sort of problem with Ubuntu that I don't have the resources to reproduce right now. 

Someone on Discord suggested:

> Click Help Menu - Toggle Developer Tools
> Click Sources - assets - index-somenumber.css
> In body, search position: fixed; -> delete this line

If you can figure out what's going wrong with ubuntu and, I'm guessing, text somehow, that would be much appreciated!

**Comment by vibl** [comment]
Removing the `position: fixed;` line solved the disappearing nodes. Thank you @abrenneke!

I have no idea why this bug would only appear in Ubuntu/Kubuntu. I guess Tauri's WebKitGTK might have some peculiarities. Or that could that be related to the X to Wayland transition, which have been affecting some aspects of Electron's rendering I believe (but they don't seem related to this). Kubuntu is still on X at the moment (I read somewhere that KDE doesn't consider Wayland mature enough).

I'll have a look into the context menu text disappearing but I don't promise anything :-)

**Comment by vibl** [comment]
The menu text disappearing in the settings modal seems to be due to `overflow-x: clip`. When I change it to `overflow-x: visible`, the text appears. Note that it has to be changed in both spans:

```html
<span class="css-oci5uq">
   <span class="css-g7kw2k" data-item-title="true">General</span>
</span>
```

**Comment by vibl** [comment]
Is there any way to have a similar CSS debugging experience in Tauri as in Chrome Devtools? The rules don't seem to be editable (so I had to add another class with an `!important` rule instead).

**Comment by abrenneke** [comment]
I don't think so, it's just using the web view's developer tools. But thank you a bunch for investigating! I'll try removing this in the main code and see how necessary they are.

**Comment by abrenneke** [comment]
Should be fixed in the upcoming 1.2.1. @all-contributors please add @vibl for bug!

**Comment by vibl** [comment]
FYI, in 1.2.1 the settings modal tabs are still invisible. The other two bugs are fixed though.
